requirements/base.txt

```diff
@@ -0,0 +1 @@
+django >= 1.6b1, < 1.9
```

Comment:
I see that you updated to 1.8 few days ago, I can make the same change here.  Looking at your Travis log, not sure why things weren't working for me >= 1.9. It's possible that I didn't have the latest version as of 5 days ago.  I'll look at this again.